src/main/java/com/davidthomasbernal/stardict/parsers/IdxParser.java

```diff
@@ -15,7 +15,7 @@
 public class IdxParser {
     private final DictionaryInfo dictionaryInfo;
     private Logger logger = Logger.getLogger(this.getClass().getName());
-    private boolean tolerateInfoMismatch = false;
```

Comment:
For clarity: Let's keep this initialization. (false may be the default value, but folks like me don't remember.)